### PR TITLE
IBP-4901 IBP-5036 save list view

### PR DESCRIPTION
**Pull request by lik-83**

### CODE REVIEW

File: src/main/jhipster/src/main/webapp/app/shared/column-filter/column-filter-dropdown-component.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import 'rxjs/add/operator/switchMap';
+import { Select2OptionData } from 'ng-select2/lib/ng-select2.interface';
+
+@Component({
+    selector: 'jhi-column-filter-dropdown',
+    template: `
+		<form #f='ngForm'>
+			<section class="filter-wrapper">
+				<div id="dropdown-container-{{this.filter.key}}">
+					<ng-select2 width="170px"
```
Comment (nahuel-soldevilla) [inline src/main/jhipster/src/main/webapp/app/shared/column-filter/column-filter-dropdown-component.ts:13]: this will come in very handy :+1:

**Review by nahuel-soldevilla — APPROVED** [review]
Awesome :+1:

File: src/main/jhipster/src/main/webapp/app/germplasm-list/germplasm-list.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.text-ellipsis {
```
[the remark this answers is not in the document]
Comment (lik-83) [reply src/main/jhipster/src/main/webapp/app/germplasm-list/germplasm-list.css]: Done!

File: src/main/jhipster/src/main/webapp/app/germplasm-list/list-columns.component.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+})
+export class ListColumnsComponent implements OnInit {
+
+    @Input() listId: number;
+
+    @Output() columnsSelectedEvent = new EventEmitter<GermplasmListColumnModel[]>();
+
+    @ViewChild('columnsDropdown') columnsDropdown: NgbDropdown;
+
+    readonly ENTRY_NO_TERM_ID = 8230;
```
[the remark this answers is not in the document]
Comment (lik-83) [reply src/main/jhipster/src/main/webapp/app/germplasm-list/list-columns.component.ts]: Done!

File: src/main/jhipster/src/main/webapp/app/germplasm-list/list.component.ts [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+
+    private getFilters() {
+        const filters = this.STATIC_FILTERS;
+        this.header.filter((value: GermplasmListObservationVariable) => this.isNotStaticColumn(value.columnCategory))
+            .forEach((column: GermplasmListObservationVariable) => {
+                const key = this.getNotStaticFilterKey(column);
+                filters[key] = {
+                    key,
+                    placeholder: 'Contains Text',
+                    type: FilterType.TEXT,
```
[the remark this answers is not in the document]
Comment (lik-83) [reply src/main/jhipster/src/main/webapp/app/germplasm-list/list.component.ts:259]: Yes! We can check if the column has a defined scale and if it's the case we can set an specific filter for that (range of numbers or whatever).

File: src/main/jhipster/src/main/webapp/app/germplasm-list/germplasm-list.scss [hunk 5]
```diff
@@ -0,0 +1,8 @@
+.column-menu {
+  width: 300px;
+}
+
+.columns-container {
+  max-height: 300px;
+  overflow: auto;
+}
```
[the remark this answers is not in the document]
Comment (lik-83) [reply src/main/jhipster/src/main/webapp/app/germplasm-list/germplasm-list.scss]: done!

**Pull request merged**